### PR TITLE
issues/1 Fix build failing.

### PR DESCRIPTION
**Pull request by pine**
Hi. I like CI's green badge 💚 

## Tasks
- [x] Update Android build tools
- [x] Update Gradle version
- [x] Supress Warnings

### CODE REVIEW

**Comment by mattak** [comment]
SUPPPEEER Great work!!

File: app/build.gradle [hunk 1]
```diff
@@ -18,6 +18,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+
+    lintOptions {
+        abortOnError false
+    }
```
Comment (mattak) [inline app/build.gradle:24]: 👀

Comment (mattak) [reply app/build.gradle:24]: I think we should be aware of lint check failure.
So fix it in another issue.

**Comment by mattak** [comment]
thanks for contribution!

**Comment by pine** [comment]
Oh, Thanks!

**Pull request merged**